multinode_integration_tests/tests/connection_termination_test.rs

```diff
@@ -86,13 +86,12 @@ fn reported_server_drop() {
     let (_, _, lcp) = mock_node
         .wait_for_package(&masquerader, Duration::from_secs(2))
         .unwrap();
-    let (stream_key, return_route_id) =
-        context_from_request_lcp(lcp, real_node.main_cryptde_null().unwrap(), &exit_cryptde);
+    let stream_key = stream_key_from_request_lcp(lcp, &exit_cryptde);
```

Comment:
Return route IDs are no longer a thing, because return-route information is now stored per stream rather than per transaction, so stream keys are used to identify it and return route IDs are eliminated.